### ISSUE TITLE
Card wrapper should be button instead of a div for a11y

### ISSUE THREAD

**Issue by I3uckwheat** [I3uckwheat/cards-of-carousal#113]
Related to: #11 

## CardWrapper.js

![image](https://user-images.githubusercontent.com/61510135/110532684-2583da00-80eb-11eb-83f5-10c049322b26.png)

Having clickable elements as divs creates accessibility issues, particularly for users using screen readers and keyboard controls. We need these clickable divs to be buttons instead.

This component also increases space taken on the bottom when activated, we should fix that at the same time here.

### Completion criteria:
- [x] Convert CardWrapper clickable div to button
- [x] Ensure styles match original div format (check hovers, mouse clicks, etc)
- [x] Ensure buttons are tab-targetable and "enter" selectable
~~Ensure cursor is a "pointer" when button is hovered.~~ <- This will be handled globally in a future issue.
- [x] Fix bottom border pushing items below it down
